src/server/ua_services_session.c

```diff
@@ -245,7 +245,7 @@ Service_ActivateSession(UA_Server *server, UA_SecureChannel *channel,
     if(session->validTill < UA_DateTime_nowMonotonic()) {
         UA_LOG_INFO_SESSION(server->config.logger, session,
                             "ActivateSession: SecureChannel %i wants "
-                            "to activate, but the session has timed out",
+                                "to activate, but the session has timed out",
```

Comment:
?

Reply:
This is a formatter bug in my IDE. I try to avoid commiting them but they slip through rather often.